### PR TITLE
bump nodejs to 6.11.0 LTS

### LINKED ISSUE #2076: Update NodeJS 6 to latest 6.11.x LTS
Update to latest NodeJS and review pre-installed packages versions

### PR DESCRIPTION
**Pull request by csantanapr**
Closes #2076

### CODE REVIEW

**Comment by csantanapr** [comment]
Need to skip 6.10.1 and pick 6.10.2 when it's out soon
https://github.com/nodejs/node/pull/12037

**Comment by csantanapr** [comment]
PG running PG1 1731

**Comment by csantanapr** [comment]
picking up nodejs 6.11

File: core/nodejsActionBase/.dockerignore [hunk 1]
```diff
@@ -1,2 +1,8 @@
 Dockerfile
 build.gradle
+.project 
+.settings 
+build.xml
```
Comment (markusthoemmes) [inline core/nodejsActionBase/.dockerignore]: I think you can safely remove this, we're not using `ant` anymore.

**Comment by csantanapr** [comment]
yeah doing the docs, forgot to push

**Comment by csantanapr** [comment]
PG3 800 🔵

**Comment by csantanapr** [comment]
I think I'm going to wait to push this and pick up this update related to security 
https://nodejs.org/en/blog/vulnerability/july-2017-security-releases/
```
The Node.js project will be releasing new versions across all of its active release lines (4.x, 6.x, 8.x) as well as 7.x the week of July 10th 2017 to incorporate a security fix.
```

**Comment by csantanapr** [comment]
Also debating how do I update the npm packages:
1. update all npm packages to latest
2. update only minor and patch levels
3. leave npm packages alone and just update nodejs

Also looking at option on creating a new kind `nodejs:6.11` and update both node and npm packages to latest, leave `nodejs:6` alone as is

**Comment by rabbah** [comment]
@csantanapr is this pr abandoned?

**Comment by csantanapr** [comment]
Updating will be done in the new repo

**Pull request closed without merging**